app/views/recipes/public_recipes.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="p-4">
+  <h1 class="text-2xl font-bold">Public Recipes</h1>
+  <% @public_recipes.each do |recipe| %>
+    <div class="shadow-lg p-4 rounded-lg bg-white mb-4">
+      <div>
+        <%= link_to recipe, class: "text-lg font-bold text-blue-500" do %>
+          <h2><%= recipe.name %></h2>
+        <% end %>
+        <p class="pb-0 text-gray-600">
+          By: <%= User.find(recipe.user_id).name  %>
+        </p>
+      </div>
+      <div>
+        <p class="text-gray-600">Total food items: <%= recipe.foods.length %></p>
+        <p class="text-gray-600">
+          Total cost: <%= recipe.foods.reduce(0) do |sum, food| %>
+            <% sum + (food.quantity * food.price)  %>
+          <% end %>
+        </p>
+      </div>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] also your public recipe page does not match the design in the wireframe, kindly fix this
![Screenshot from 2023-02-18 17-32-00](https://user-images.githubusercontent.com/66097371/219877179-806a7d0b-17c8-4ec5-9936-5bea0b9e3f15.png)